standard/statements.md

```diff
@@ -606,6 +615,8 @@ Local function bodies are always reachable. The endpoint of a local function dec
 
 If the type of the argument to a local function is `dynamic`, the function to be called shall be resolved at compile time, not runtime.
 
+A local function is treated like a method, except in regard to its captured variables. The initial state of a captured variable inside a local function is the intersection of the nullable state of the variable at all the uses of that local function. A use of a local function is either a call to that function, or where it is converted to a delegate.
```

Comment:
We need to define how "intersection" is calculated here. May say the initial state of the capture variable is not-null if and only if the state of the captured variable is not-null at all uses of that local function?

Reply:
Do not all three states need to be considered? This might give:

- If at all uses the null state of a captured variable is “not null” then the initial state is “not null”
- Otherwise if at any of the uses the null state is “maybe default” then the initial state is “maybe default”
- Otherwise the initial state is “maybe null”

But that is a *guess* at this time, particularly as the definition and propagation of the “maybe” states probably requires some clarification.